site/docs/install-compile-source.md

```diff
@@ -154,7 +155,7 @@ your PATH so that you can run "bazel" everywhere.)
 
 You can also build Bazel from scratch, without using an existing Bazel binary.
```

Comment:
Bootstrap should move to a separate page. 1 out of 1000 bazel users should ever to that.
Even building from source should be a separate page, since that is a non-preferred path and most people won't do that.